src/main/org/deidentifier/arx/io/ImportConfigurationJDBC.java

```diff
@@ -1,205 +1,237 @@
-        ResultSet rs = null;
-        int index = -1;
-        try {
-            
-            rs = connection.getMetaData().getColumns(null,
-                                                     null,
-                                                     table,
-                                                     null);
-            
-            int i = 0;
-            while (rs.next()) {
-                if (rs.getString("COLUMN_NAME").equals(aliasName)) {
-                    index = i;
-                }
-                i++;
-            }
-        } catch (SQLException e) {
-            /* Catch silently */
-        } finally {
-            try {
-                if (rs != null) {
-                    rs.close();
-                }
-            } catch (SQLException e) {
-                /* Ignore silently */
-            }
-        }
-        if (index != -1) {
-            return index;
-        }
-        throw new NoSuchElementException("Index for column '" + aliasName +
-                                         "' couldn't be found");
-        
-    }
-    
-    /**
-     * @return {@link #connection}
-     */
-    protected Connection getConnection() {
-        return this.connection;
-    }
-    
-    /**
-     * @return {@link #table}
-     */
-    protected String getTable() {
-        return this.table;
-    }
-    
-    /**
-     * Returns whether we need to close the connection
-     * @return
-     */
-    protected boolean isManageConnection() {
-        return this.manageConnection;
-    }
-}
+/*
+ * ARX: Powerful Data Anonymization
+ * Copyright 2014 - 2015 Karol Babioch, Fabian Prasser, Florian Kohlmayer
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.deidentifier.arx.io;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.NoSuchElementException;
+
+/**
+ * Configuration describing a JDBC source.
+ *
+ * @author Karol Babioch
+ * @author Fabian Prasser
+ */
+public class ImportConfigurationJDBC extends ImportConfiguration {
+
+    /**
+     * Connection to be used.
+     *
+     * @see {@link #setConnection(Connection)}
+     * @see {@link #getConnection()}
+     */
+    private Connection connection;
+
+    /**
+     * Name of table to be used.
+     *
+     * @see {@link #setTable(String)}
+     * @see {@link #getTable()}
+     */
+    private String table;
+
+    public String getSchemaName() {
```

Comment:
Please add Javadoc comments to all methods added to this class and to the fields.